### ISSUE TITLE
Missing explicit from data

### ISSUE THREAD

**Issue by domialex** [domialex/Sidekick#520]
Somehow I can't find this attribute in the API: `You have Shocking Conflux for 3 seconds every 8 seconds`

```
Rarity: Rare
Blight Guardian
Hunter Hood
--------
Evasion Rating: 231 (augmented)
--------
Requirements:
Level: 64
Dex: 87
--------
Sockets: G 
--------
Item Level: 80
--------
Adds 28 to 51 Fire Damage to Spells
+28 to Evasion Rating
+47 to maximum Life
11% increased Rarity of Items found
+29% to Cold Resistance
You have Shocking Conflux for 3 seconds every 8 seconds
--------
Hunter Item
```
![image](https://user-images.githubusercontent.com/4694217/106218503-652edc00-61a5-11eb-9ef5-27103370a738.png)

**Comment by domialex** [comment]
Ah, it seems it's an exception in their data:
![image](https://user-images.githubusercontent.com/4694217/106218612-9c9d8880-61a5-11eb-97b9-6de1389630bb.png)

We need to select the correct value from the explicit `explicit.stat_1190121450`

**Comment by leMicin** [comment]
Fixed in 2.0
